src/qp/metrics/point_estimate_metric_classes.py

```diff
@@ -0,0 +1,172 @@
+import numpy as np
+from qp.metrics.base_metric_classes import (
+    MetricOuputType,
+    PointToPointMetric,
+)
+
+
+class PointStatsEz(PointToPointMetric):
+    """Copied from PZDC1paper repo. Adapted to remove the cut based on
+    magnitude."""
+
+    metric_name = "point_stats_ez"
+
+    #! This doesn't seem quiet correct, perhaps we need a `single_value_per_input_element` ???
+    metric_output_type = MetricOuputType.one_value_per_distribution
```

Comment:
Not sure if this is the correct MetricOutputType or not.

Reply:
Yes, I think it is.